$NetBSD: patch-aa,v 1.1.1.1 2010/07/11 22:41:59 seb Exp $

Prevent a prototype error between File::chmod::chmod() and CORE::chmod()

--- lib/Git/CPAN/Patch/Import.pm.orig	2010-06-08 01:03:43.000000000 +0000
+++ lib/Git/CPAN/Patch/Import.pm
@@ -13,7 +13,7 @@ use autodie;
 use Archive::Extract;
 $Archive::Extract::PREFER_BIN = 1;
 
-use File::chmod;
+use File::chmod qw();
 use File::Find;
 use File::Basename;
 use File::Spec::Functions;
@@ -47,9 +47,9 @@ sub cpanplus {
 sub _fix_permissions {
     my $dir = shift;
 
-    chmod "u+rx", $dir;
+    File::chmod::chmod "u+rx", $dir;
     find(sub {
-        -d $_ ? chmod "u+rx", $_ : chmod "u+r", $_;
+        -d $_ ? File::chmod::chmod "u+rx", $_ : File::chmod::chmod "u+r", $_;
     }, $dir);
 }
 
@@ -584,4 +584,4 @@ as a function so git-backpan-init goes f
 
 =cut
 
-1;
\ No newline at end of file
+1;
